services/ui-src/src/utils/PackageApi.js

```diff
@@ -71,10 +71,10 @@ class PackageApi {
       );
       throw new Error("Missing required data or uploads");
     }
-    console.log("componentType: ", data.type);
-    console.log("posting to: ", SUBMIT_API_CALL[data.type]);
+    console.log("componentType: ", componentType);
+    console.log("posting to: ", SUBMIT_API_CALL[componentType]);
```

Comment:
At some point, I do envision the SUBMIT_API_CALL being a passed variable stored in the OneMACFormConfig

Reply:
> At some point, I do envision the SUBMIT_API_CALL being a passed variable stored in the OneMACFormConfig

Agreed, i keep wanting to refactor that portion, and it keeps getting usurped by other priorities. :)